install.sh

```diff
@@ -1,23 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
not used anymore by the formula https://github.com/elifesciences/profiles-formula/search?utf8=%E2%9C%93&q=install.sh&type=